readme.txt

```diff
@@ -119,6 +119,7 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [4.10.2] TBD =
 
+* Fix - Use a md5 hash for checkbox and radio option ids to prevent fields from not saving if they a large amount of characters [119448]
```

Comment:
```suggestion
* Fix - Use a md5 hash for checkbox and radio option names to prevent fields from not saving if they a large amount of characters [119448]
```